src/Microsoft.MobileBlazorBindings.Authentication/Models/OidcAuthenticationState.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.MobileBlazorBindings.Authentication
+{
+    public class OidcAuthenticationState
+    {
+        //
+        // Summary:
+        //     Gets or sets the start URL.
+        //
+        // Value:
+        //     The start URL.
```

Comment:
Could you replace that with valid xml docs?
(here and other places)

Reply:
Done